<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialportWidgetClass</class>
 <widget class="QWidget" name="SerialportWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serialport</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <item row="1" column="1">
      <widget class="QComboBox" name="baud_box">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>56000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>78600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>230400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>460800</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>BaudRate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="serial_label">
       <property name="text">
        <string>SerialPort</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="port_box">
         <item>
          <property name="text">
           <string>com1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>com2</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="refresh_btn">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Bytes</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="send_size">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="start_btn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
